data/yaml/joint.yaml

```diff
@@ -1284,6 +1284,9 @@ storynlp:
 stylevar:
 - W17-49
 - W18-16
+syntaxfest:
```

Comment:
@mjpost added 2021 and 2019 according to its website https://syntaxfest.github.io/